projects/step-frontend/src/lib/modules/timeseries/execution-page/time-series-keywords.context.ts

```diff
@@ -14,12 +14,16 @@ export class TimeSeriesKeywordsContext {
   private keywordToggleSubject = new Subject<KeywordSelection>();
   private selectAllSubject = new BehaviorSubject<boolean>(true);
 
-  private colorsPool: TimeseriesColorsPool;
+  private readonly colorsPool: TimeseriesColorsPool;
 
   constructor(colorsPool: TimeseriesColorsPool) {
     this.colorsPool = colorsPool;
   }
 
+  getColorsPool() {
+    return this.colorsPool;
```

Comment:
Remove this method, as field is already readonly

Reply:
@lucianbaciu we had a discussion about our coding conventions for setter / getter and decided to use public properties or the set/get syntax of TS. For this case we already have the exposed readonly property so we don't need anything else.

Reply:
@neogucky alright. I will check for other getter/setters